packages/expo-video/ios/Records/VideoSource.swift

```diff
@@ -3,9 +3,15 @@
 import Foundation
```

Comment:
In general we almost never need to import Foundation, it's implicitly auto-imported